### PR TITLE
Add scattersize

### PR DESCRIPTION
**Pull request by rhpvorderman**
### Checklist
- [x] Pull request details were added to CHANGELOG.md
- [x] Documentation was updated (if required).

### CODE REVIEW

File: tests/integration/Germline_PairedEnd.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+"Germline.outputDir":"/home/rhpvorderman/PycharmProjects/germline-DNA/test-output",
```
Comment (JasperBoom) [inline tests/integration/Germline_PairedEnd.json]: Is this a correct output directory?

File: somatic.wdl [hunk 2]
```diff
@@ -249,6 +252,10 @@ workflow Somatic {
 
         dockerImagesFile: {description: "A YAML file describing the docker image used for the tasks. The dockerImages.yml provided with the pipeline is recommended.",
                            category: "advanced"}
+        scatterSize: {description: "The size of the scattered regions in bases for the GATK subworkflows. Scattering is used to speed up certain processes. The genome will be seperated into multiple chunks (scatters) which will be processed in their own job, allowing for parallel processing. Higher values will result in a lower number of jobs. The optimal value here will depend on the available resources.",
+              category: "advanced"}
+        scatterSizeMillions:{ description: "Same as scatterSize, but is multiplied by 1000000 to get scatterSize. This allows for setting larger values more easily",
```
Comment (JasperBoom) [inline somatic.wdl]:
```suggestion
        scatterSizeMillions:{ description: "Same as scatterSize, but is multiplied by 1000000 to get scatterSize. This allows for setting larger values more easily.",
```

File: germline.wdl [hunk 3]
```diff
@@ -179,9 +162,12 @@ workflow Germline {
...
         runSVcalling: {description: "Whether or not Structural-variantcalling should be run.", category: "advanced"}
         runMultiQC: {description: "Whether or not MultiQC should be run.", category: "advanced"}
-        normalizedVcf: {description: "Normalize the multisample.", category: "common"}
         XNonParRegions: {description: "Bed file with the non-PAR regions of X.", category: "common"}
         YNonParRegions: {description: "Bed file with the non-PAR regions of Y.", category: "common"}
+        scatterSize: {description: "The size of the scattered regions in bases for the GATK subworkflows. Scattering is used to speed up certain processes. The genome will be seperated into multiple chunks (scatters) which will be processed in their own job, allowing for parallel processing. Higher values will result in a lower number of jobs. The optimal value here will depend on the available resources.",
+              category: "advanced"}
+        scatterSizeMillions:{ description: "Same as scatterSize, but is multiplied by 1000000 to get scatterSize. This allows for setting larger values more easily",
```
Comment (JasperBoom) [inline germline.wdl]:
```suggestion
        scatterSizeMillions:{ description: "Same as scatterSize, but is multiplied by 1000000 to get scatterSize. This allows for setting larger values more easily.",
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,6 +11,7 @@ that users understand how the changes affect the new version.
 
 version 3.0.0-dev
 -----------------
++ Add scatterSize option to centrally control the scatter size
```
Comment (JasperBoom) [inline CHANGELOG.md]: Should the removing of normalization be mentioned in the CHANGELOG?

Comment (rhpvorderman) [reply CHANGELOG.md]: Well, the adding was not... And it was not present in  the previous version.

File: tests/integration/Germline_PairedEnd.json (hunk 1 above)
Comment (rhpvorderman) [reply tests/integration/Germline_PairedEnd.json]: good catch. Don't know why it did this?

Comment (rhpvorderman) [reply tests/integration/Germline_PairedEnd.json]: Whoops, seems I run a `sed` command in the root directory.

File: CHANGELOG.md (hunk 4 above)
Comment (JasperBoom) [reply CHANGELOG.md]: Ah, then it makes sense not that mention it.

**Pull request merged**